src/test/skript/tests/syntaxes/effects/EffBlockUpdate.sk

```diff
@@ -0,0 +1,8 @@
+test "block update":
+	set {_loc} to spawn of world "world" ~ vector(10,10,10)
+	set {_blocks::*} to blocks in radius 2 of block at {_loc}
+
+	force update blocks at {_blocks::*} as sand without physics updates
+	assert blocks at {_blocks::*} is sand with "As: 1 or more block state did not update with or without physics"
+	force update blocks at {_blocks::*} as air without physics updates
```

Comment:
just saw your change in the last commit, dont forget to update your tests
```suggestion
	update blocks at {_blocks::*} as sand without physics updates
	assert blocks at {_blocks::*} is sand with "As: 1 or more block state did not update with or without physics"
	update blocks at {_blocks::*} as air without physics updates
```

Reply:
Damn, thank you lol